ivy/functional/ivy/layers.py

```diff
@@ -2107,53 +2104,103 @@ def conv(
-    filters
-        Convolution filters *[fd,fh,fw,d_in/feature_group_count,d_out]*.
-    strides
+    x : ivy.Array
+        Input volume with shape [batch_size, depth, height, width, channels].
+    filters : ivy.Array
+        Convolution filters with shape [fd, fh, fw, d_in, d_out].
+    strides : Tuple[int, int, int]
         The stride of the sliding window for each dimension of input.
-    padding
-        either the string ‘SAME’ (padding with zeros evenly), the string ‘VALID’ (no
-        padding), or a sequence of n (low, high) integer pairs that give the padding to
-        apply before and after each spatial dimension.
-    transpose
-        True for computing transpose convolution, and False for dilated convolution.
-        When True, `x_dilations` must be 1 (the default).
-    dims
-        Either 1, 2, or 3 corresponding to 1-D, 2-D, and 3-D convolution.
-    output_shape
-        Shape of the output (Default value = None)
-    data_format
-        Either "channel_first" or "channel_last". "channel_first" corresponds to "NCW",
-        "NCHW", "NCDHW" input data formatS for 1-D, 2-D, 3-D convolution respectively,
-        while "channel_last" corresponds to "NWC", "NHWC", "NDHWC" respectively.
-    filter_format
-        Either "channel_first" or "channel_last". "channel_first" corresponds to "OIW",
-        "OIHW", "OIDHW" input data formats for 1-D, 2-D, 3-D convolution respectively,
-        while "channel_last" corresponds to "WIO", "HWIO", "DHWIO" respectively.
-    feature_group_count
-         split input into groups, d_in should be divisible by the number of groups.
-         (Default value = 1)
-    x_dilations
-        The dilation factor for each dimension of input. (Default value = 1)
-    dilations
-        The dilation factor for each dimension of input. (Default value = 1)
-    bias
-        Bias array of shape *[d_out]*.
-    out
-        optional output array, for writing the result to. It must have a shape that the
-        inputs broadcast to.
+    padding : str
+        Either 'SAME' (padding so that the output's shape is the same as the
+        input's) or 'VALID' (padding so that the output's shape is `output_shape`).
+    output_shape : Optional[ivy.Shape], optional
+        Shape of the output, by default None.
+    data_format : str, optional
+        Ordering of the dimensions in the input, 'NDHWC' or 'NCDHW', by default 'NDHWC'.
+    dilations : Tuple[int, int, int], optional
+        The dilation factor for each dimension of input, by default (1, 1, 1).
+    bias : Optional[ivy.Array], optional
+        Bias array of shape [d_out], by default None.
+    out : Optional[ivy.Array], optional
+        Optional output array for writing the result to.
 
     Returns
     -------
-    ret
-        The result of the transpose or dilated convolution operation.
+    ivy.Array
+        The result of the transpose convolution operation.
+
+    Examples
```

Comment:
examples addition is correct, also please make sure the returns are identical in the doc strings

Reply:
Thank you, I have also checked the returns are identical in the doc strings.